### PR TITLE
Fixed duplicate reports of SHARE_URL and SOCIAL_WIDGET loading

### PR DESCRIPTION
**Pull request by daviddahl**

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -470,6 +474,11 @@ let NetworkListener = {
     if (!aHttpChannel.URI || !(aHttpChannel.URI.host in kSOCIAL_HOSTS)) {
       return;
     }
+    // Ignore all other HTTP activity except the request header which will have
+    // the social urls we are looking for in them
+    if (aActivitySubtype != this.activityTypes.ACTIVITY_SUBTYPE_RESPONSE_COMPLETE) {
```
Comment (monicachew) [inline lib/main.js]: you should be able to use Ci.nsIHttpActivityObserver.ACTIVITY_SUBTYPE_RESPONSE_COMPLETE

File: lib/main.js [hunk 2]
```diff
@@ -69,6 +71,8 @@ const kSHARE_URLS = [
 const kSOCIAL_WIDGET_URLS = [
   "http://connect.facebook.net/en_US/all.js",
   "https://connect.facebook.net/en_US/all.js",
+  "http://www.facebook.com/plugins/like.php",
+  "https://www.facebook.com/plugins/like.php",
```
Comment (monicachew) [inline lib/main.js]: I think you can trim the scheme for all of these and it will still work

**Comment by monicachew** [comment]
Thanks for working on this. Does this link help?

https://developers.google.com/+/web/+1button/

File: lib/main.js (hunk 1 above)
Comment (daviddahl) [inline lib/main.js]: will do! thanks!

File: lib/main.js (hunk 2 above)
Comment (daviddahl) [inline lib/main.js]: yeah. I will do that. soon.

**Comment by daviddahl** [comment]
Yes, that helps!

File: lib/main.js [hunk 3]
```diff
@@ -506,16 +511,25 @@ let NetworkListener = {
...
+  activityTypes: {
+    ACTIVITY_TYPE_SOCKET_TRANSPORT: 0x0001,
+    ACTIVITY_TYPE_HTTP_TRANSACTION: 0x0002,
+    ACTIVITY_SUBTYPE_REQUEST_HEADER: 0x5001,
+    ACTIVITY_SUBTYPE_REQUEST_BODY_SENT: 0x5002,
+    ACTIVITY_SUBTYPE_RESPONSE_START: 0x5003,
+    ACTIVITY_SUBTYPE_RESPONSE_HEADER: 0x5004,
+    ACTIVITY_SUBTYPE_RESPONSE_COMPLETE: 0x5005,
+    ACTIVITY_SUBTYPE_TRANSACTION_CLOSE: 0x5006
+  }
```
Comment (monicachew) [inline lib/main.js:532]: Now this block is not needed

**Pull request merged**